### PR TITLE
Backend for angular frontend

### PR DESCRIPTION
**Pull request by rdvhoorn**

### CODE REVIEW

File: models/companyOpportunity.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    /**
+     * Level of education needed for the opportunity
+     */
+    educationLevel: Sequelize.STRING,
+
+    /**
+     * Category of the opportunity
+     */
+    category: Sequelize.STRING
```
Comment (HiddeKoerts) [inline models/companyOpportunity.js:43]: Should we enforce some restrictions here? As in make a enum or something?

File: routes/activities.js [hunk 2]
```diff
@@ -215,11 +211,11 @@ router.route("/pictures/:id")
      * Edits a picture
      */
     .put(function (req, res, next) {
-        // delete old picture
+        // // delete old picture
```
Comment (HiddeKoerts) [inline routes/activities.js:214]:
```suggestion
        // delete old picture
```

File: routes/partners.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+router.route("/internships")
+    .get(function (req, res) {
+        /**
+         * Route for getting all internships from the database.
+         */
+        var user = res.locals.session ? res.locals.session.user : null;
+        permissions.check(user, {type: "INTERNSHIP_VIEW"}).then(function (result) {
+            if (!result) {
+                res.status(403).send("You are not allowed to get all internships");
```
Comment (HiddeKoerts) [inline routes/partners.js:16]:
```suggestion
                res.status(403).send("You do not have the permissions to view internships");
```

**Review by HiddeKoerts — APPROVED** [review]

File: models/companyOpportunity.js (hunk 1 above)
Comment (rdvhoorn) [reply models/companyOpportunity.js:43]: yeah but we should check that later anyway cuz that has to happen for the whole database etc

**Review by HiddeKoerts — APPROVED** [review]

**Pull request merged**